gcomm/src/pc_proto.hpp

```diff
@@ -244,6 +244,7 @@ class gcomm::pc::Proto : public Protolay
     void handle_user(const Message&, const Datagram&,
                      const ProtoUpMeta&);
     void deliver_view(bool bootstrap = false);
+    void identity_changed(const gcomm::UUID& old_identity, const gcomm::UUID& new_identity);
```

Comment:
The below version is more readable
```suggestion
    void identity_changed(const gcomm::UUID& old_identity, 
                          const gcomm::UUID& new_identity);
```

Reply:
done